test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usda

```diff
@@ -0,0 +1,312 @@
+#usda 1.0
+(
+    customLayerData = {
+        dictionary cameraSettings = {
+            dictionary Front = {
+                double3 position = (0, 0, 500)
+                double radius = 500
+            }
+            dictionary Perspective = {
+                double3 position = (-4.556, 5.556, -4.556)
+                double3 target = (0.5, 0.5, 0.5)
+            }
+            dictionary Right = {
+                double3 position = (-500, 0, 0)
+                double radius = 500
+            }
+            dictionary Top = {
+                double3 position = (0, 500, 0)
+                double radius = 500
+            }
+            string boundCamera = "/Camera"
+        }
+        dictionary renderSettings = {
+            double "rtx:sceneDb:ambientLightIntensity" = 0
+        }
+    }
+    defaultPrim = "Block_Test_World"
+    metersPerUnit = 1
```

Comment:
Ah, I should remove this line, for the reasons detailed here: https://github.com/erich666/McUsd/issues/3 - a rough area in the USD spec, better avoided, since that is not what this test is about.